### ISSUE TITLE
Tapping above or below the ActionBar (in content area or in status bar) will register the tap

### ISSUE THREAD

**Issue by pentool** [NativeScript/NativeScript#5363]
## Please, provide the details below:
It seems that when you have ActionItems in the ActionBar (either as text or as icons), the tap area is extending above and below the ActionBar. Eg, you can tap above an ActionItem in the status bar (iOS), or you can tap about 5 pixels below the ActionBar (in the content area), and the tap will register (provided you have a tap handler on the ActionItem).

### Which platform(s) does your issue occur on?
iOS. Have not tested with Android.

### Please provide the following version numbers that your issue occurs with:
- CLI: 3.4.1

### Please tell us how to recreate the issue in as much detail as possible.
The only reason I'm not providing a playground sample is because this is best seen/experienced on a simulator where you can click using the mouse.

1. Create a bare bones app using a template of your choice. Alternatively, you could test it on an existing project.
2. Put an ActionItem into the ActionBar. Use the ActionItem either as a text or as an icon.
3. Place the mouse about 5 or so pixels BELOW the ActionBar (which is the content area of your app) and click. The tap handler for the ActionItem will be executed.
4. You can repeat the same thing, but this time click above the ActionItem, in the status bar (iOS) area. Again, the tap handler will be executed.

It seems the tap area is extending a few pixels above and below the ActionBar.

**Comment by tsonevn** [comment]
I tested this behavior on my side, however, it seems that this is expected behavior and the tap event will be triggered also to the area around the ActionItem. Keep in mind that the tap event will still trigger inside the ActionBar. Also, I tested this case on native iOS project build with Xcode. For your convenience, I am attaching GIF file which demonstrates the behavior of a native project.

![screencast 2018-01-31 at 9 21 28 am](https://user-images.githubusercontent.com/17448734/35610391-c2275e2c-0669-11e8-8316-b7c9f0572d24.gif)

Something more, the ActionBar component is a common abstraction of iOS Navigation View controller and we just implement the functionality provided by the native component.

**Comment by pentool** [comment]
Thanks for the test, but I think you misunderstood - or I explained it wrong.

In the provided GIF, you are clicking right above and below the ActionItem. That is **not** where you supposed to click.

There's a gray horizontal line right below the ActionBar (or Navigation Bar). You need to click **below** that, about 5px or so. That's when the tap event is still registering.

Regarding the status bar. The status bar is at the very top of the display that shows the time, battery status, wi-fi strength, etc. What I meant is, that when you click **inside** the status bar, the tap event is still getting registered.

Please look at the attached screen recording. (Sorry, I'm only showing the top part of the simulator for the content area is top secret :-) There are 3 icons in the ActionBar. Each icon has a tap event attached to it in the XML.

In the screen recording I'm showing that I don't have to click the icons. The 3 color bars are BELOW the ActionBar. You can see I'm clicking below the ActionBar and the tap event is registered - as if I was clicking on the icon. Also, I'm clicking in the status bar where the signal and battery strength indicators are and the tap still registers. As if the tap area is extended above and below the icons.

It seems like I'm not able to attach a .mov file, so here's a link to dropbox:
https://www.dropbox.com/s/24iei6y6u3bx3q3/tap.mov?dl=0

**Comment by tsonevn** [comment]
Hi @pentool,
Thank you for the sample video.
However, we are still unable to recreate this behavior on our side. Regarding that, please review the below-attached sample project and make the needed changes, which will allow us to recreate the problem.

Thank you in advance for your cooperation.
[Archive.zip](https://github.com/NativeScript/NativeScript/files/1681014/Archive.zip)

**Comment by pentool** [comment]
Thanks for looking at this issue and providing the sample project @tsonevn !
I have downloaded the archive, however I did not have to make any adjustment to it. I was able to reproduce it with the provided project as-is. I'm attaching the screen recording via dropbox.

You will be able to see where I'm clicking by the indicated mouse click circles. Notice that I can simply click in the statusbar or below the navbar to activate the "Right" button. It behaves the same way on my device, too.

iTouch 5G / iOS 9.3.5
Xcode 8.2.1
OS X 10.11.6
CLI 3.4.1

Dropbox link: https://www.dropbox.com/s/edcw44ms5f0lt95/tap2.mov?dl=0

**Comment by EddyVerbruggen** [comment]
You are all right: the problem occurs, and it doesn't. It depends on the iOS version 😁 - see the video.

iPhone 5s, running the exact same app:
- left = iOS 10.3.1: tap area too large
- right = iOS 11.2: tap area OK

![clickdiff](https://user-images.githubusercontent.com/1426370/35776232-00011dd2-0999-11e8-9d61-47cb5c7315e9.gif)

**Comment by CatchABus** [comment]
Closing as the problem was related to older OS versions.
